PACKAGE = "cartesian_impedance_controller"
    
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()



gen.add("f_x", double_t, 0, "Apply a force in the x-direction",    0, -10,   10)
gen.add("f_y", double_t, 0, "Apply a force in the y-direction",    0, -10,   10)
gen.add("f_z", double_t, 0, "Apply a force in the z-direction",    0, -10,   10)
gen.add("tau_x", double_t, 0, "Apply a torque  around the x direction",    0, -10,   10)
gen.add("tau_y", double_t, 0, "Apply a torque  around the y direction",    0, -10,   10)
gen.add("tau_z", double_t, 0, "Apply a torque  around the z direction",    0, -10,   10)

gen.add("apply_wrench", bool_t, 0, "Apply Cartesian wrench.",   False)

exit(gen.generate(PACKAGE, "cartesian_impedance_controller", "wrench_config"))
